src/components/OwnerPanel/OwnerPanel.tsx

```diff
@@ -138,8 +182,8 @@ const OwnerPanel = (props: Props) => {
           })}
         </div>
       ) : (
-        ''
-      )}
+          ''
+        )}
```

Comment:
is this right? did you run `yarn format` and `yarn eslint --fix .`

Reply:
just ran it, thanks for the catch!